packages/expo-dev-client/android/src/androidTest/java/com/expo/modules/devclient/activities/DevClientBundledAppActivity.kt

```diff
@@ -0,0 +1,39 @@
+package com.expo.modules.devclient.activities
+
+import android.content.Intent
+import com.facebook.react.ReactActivityDelegate
+import com.facebook.react.ReactNativeHost
+import expo.modules.devlauncher.DevLauncherController.Companion.tryToHandleIntent
+import expo.modules.devlauncher.DevLauncherController.Companion.wrapReactActivityDelegate
+import expo.modules.devlauncher.launcher.DevLauncherReactActivityDelegateSupplier
+import expo.modules.devmenu.react.DevMenuAwareReactActivity
+
+internal var reactNativeHostHolder: ReactNativeHost? = null
+
+internal class DevClientBundledAppActivity : DevMenuAwareReactActivity() {
```

Comment:
is this class something that will need to be kept up to date if/when we make changes to how DevMenuAwareReactActivity is integrated into projects?

Reply:
Yes, we will need to update it whenever the installation steps change. However, we can't do anything about it 🤷‍♂️